documents/refund.md

```diff
@@ -193,27 +195,27 @@ body, err := client.Payment.All(option, nil)
 **Response:**
 ```json
 {
```

Comment:
add comma "count" : 1**,**

Reply:
> add comma "count" : 1**,**

[fixed syntax](https://github.com/razorpay/razorpay-go/blob/refund_doc_correction/documents/refund.md?plain=1#L181)

Reply:
> with **"speed": "optimum",** returns null

If you are referring to `Create an instant refund` then use this snippet with my api key
`	data := map[string]interface{}{
		"speed": "optimum",
		"receipt": "#Receipt No. 35",
	}
	body, err := client.Payment.Refund("pay_JLcbRwKkgDlWWJ",100, data, nil)`